### PR TITLE
Convert omitempty boolean params to pointers

### PR DESCRIPTION
**Pull request by sverdlov93**
Signed-off-by: sverdlov93 <sverdlov93@gmail.com>

- [ ] All [tests](https://github.com/jfrog/jfrog-client-go#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [ ] This pull request is on the dev branch.
- [ ] I used gofmt for formatting the code before submitting the pull request.
-----
Convert omitempty boolean params to pointers
Also. added tests to verify that assigning false/true to boolean params works.

Following #484 and #462

### CODE REVIEW

File: tests/utils_test.go [hunk 1]
```diff
@@ -131,18 +131,18 @@ const (
...
 	TestRepository = flag.Bool("test.repository", false, "Test repositories in Artifactory")
-	RtUrl = flag.String("rt.url", "", "Artifactory url")
+	RtUrl = flag.String("rt.url", "https://ecosysjfrog.jfrog.io/artifactory/", "Artifactory url")
 	DistUrl = flag.String("ds.url", "", "Distribution url")
 	XrayUrl = flag.String("xr.url", "", "Xray url")
 	PipelinesUrl = flag.String("pipe.url", "", "Pipelines url")
-	RtUser = flag.String("rt.user", "", "Artifactory username")
-	RtPassword = flag.String("rt.password", "", "Artifactory password")
+	RtUser = flag.String("rt.user", "michaelsv", "Artifactory username")
+	RtPassword = flag.String("rt.password", "Jfrog-12", "Artifactory password")
```
Comment (joeh90) [inline tests/utils_test.go:145]: Guessing these values were left in accidentally?

File: tests/utils_test.go [hunk 2]
```diff
@@ -131,18 +131,18 @@ const (
...
 	ciRunId = flag.String("ci.runId", "", "A unique identifier used as a suffix to create repositories in the tests")
-	TestArtifactory = flag.Bool("test.artifactory", false, "Test Artifactory")
+	TestArtifactory = flag.Bool("test.artifactory", true, "Test Artifactory")
 	TestDistribution = flag.Bool("test.distribution", false, "Test distribution")
 	TestXray = flag.Bool("test.xray", false, "Test xray")
 	TestPipelines = flag.Bool("test.pipelines", false, "Test pipelines")
 	TestAccess = flag.Bool("test.access", false, "Test access")
 	TestRepository = flag.Bool("test.repository", false, "Test repositories in Artifactory")
-	RtUrl = flag.String("rt.url", "", "Artifactory url")
+	RtUrl = flag.String("rt.url", "https://ecosysjfrog.jfrog.io/artifactory/", "Artifactory url")
```
Comment (joeh90) [inline tests/utils_test.go:140]: same as below

Comment (sverdlov93) [reply tests/utils_test.go]: Removed. Thanks for noticing @joeh90

**Review by eyalbe4 — CHANGES_REQUESTED** [review]
We may also need to change the relevant README example following this change.

**Review by eyalbe4 — APPROVED** [review]

**Pull request merged**